app/views/products/show.html.erb

```diff
@@ -23,10 +23,17 @@
       <% end %>
     </div>
   </div></br>
+</div
 <div class= "container">
   <% if @product.photos.attached? %>
     <% @product.photos.each do |photo| %>
       </br><%= image_tag photo, width: 1000, height: 700 %></br>
     <% end %>
   <% end %>
-</div> 
```

Comment:
ma'am I am not getting, why it showing this "no new line at the end of file" here